rules/60-sensor.rules

```diff
@@ -0,0 +1,10 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="sensor_end"
+KERNEL!="i2c*", GOTO="sensor_end"
```

Comment:
I would like to match on iio devices instead, but they don't have a modalias themselves. If there was a way to match the parent device instead.

This is what an IIO device's tree might look like when not connected via I2C:
```
P: /devices/pci0000:00/0000:00:14.0/usb3/3-1/3-1:1.0/0003:273F:1008.001A
E: DEVPATH=/devices/pci0000:00/0000:00:14.0/usb3/3-1/3-1:1.0/0003:273F:1008.001A
E: DRIVER=hid-sensor-hub
E: HID_ID=0003:0000273F:00001008
E: HID_NAME=Hughski Ltd. ColorHugALS
E: HID_PHYS=usb-0000:00:14.0-1/input0
E: HID_UNIQ=
E: MODALIAS=hid:b0003g0003v0000273Fp00001008
E: SUBSYSTEM=hid

P: /devices/pci0000:00/0000:00:14.0/usb3/3-1/3-1:1.0/0003:273F:1008.001A/HID-SENSOR-200041.2.auto
E: DEVPATH=/devices/pci0000:00/0000:00:14.0/usb3/3-1/3-1:1.0/0003:273F:1008.001A/HID-SENSOR-200041.2.auto
E: DEVTYPE=mfd_device
E: DRIVER=hid_sensor_als
E: MODALIAS=platform:HID-SENSOR-200041
E: SUBSYSTEM=platform

P: /devices/pci0000:00/0000:00:14.0/usb3/3-1/3-1:1.0/0003:273F:1008.001A/HID-SENSOR-200041.2.auto/iio:device0
N: iio:device0
E: DEVNAME=/dev/iio:device0
E: DEVPATH=/devices/pci0000:00/0000:00:14.0/usb3/3-1/3-1:1.0/0003:273F:1008.001A/HID-SENSOR-200041.2.auto/iio:device0
E: DEVTYPE=iio_device
E: IIO_SENSOR_PROXY_TYPE=iio-buffer-als
E: MAJOR=240
E: MINOR=0
E: SUBSYSTEM=iio
E: SYSTEMD_WANTS=iio-sensor-proxy.service
E: TAGS=:systemd:
E: USEC_INITIALIZED=98015640898
```